src/main/java/core/basesyntax/service/operations/Expense.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.db.Storage;
+
+public class Expense implements OperationService {
```

Comment:
let's rename OperationService to OperationHandler and Expense to ExpenseOperationHandler to keep our naming consistent and stick to the strategy pattern
do the same for other implementations